index.html

```diff
@@ -329,6 +339,83 @@ <h1 class="font-crete-round text-xl primary-text">About me</h1>
           </ul>
         </div>
       </section>
+      <section class="contact-section bg-contact-color" id="contact">
+        <div class="contact-container d-flex">
+          <h2 class="font-inter font-center contact-para">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form class="contacts-form d-flex" action="#" method="POST">
```

Comment:
- [ ] Kindly update form with the correct action, i.e where the form will be sent 

_it's a formspree link something like `https://formspree.io/f/{form_id}`_